crates/sui-core/src/authority.rs

```diff
@@ -3315,6 +3345,29 @@ impl AuthorityState {
         }
     }
 
+    #[instrument(level = "trace", skip_all)]
+    pub fn get_transactions_and_serialized_sizes(
```

Comment:
This function seems to belong to the ExecutionCacheRead trait

Reply:
moved